homework-2/utube/src/components/video_preview.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+
+export class VideoPreview extends Component {
+    render () {
+        const preview = this.props.preview;
+        return (
+            <li className="list-group-item" onClick={() => this.props.activatePreview(preview)}>
+                <div className="video-list media">
+                    <div className="video-list media">
+                        <div className="media-left">
+                            <a href={this.props.makeActive}><img alt="" className="media-object" src={preview.thumbnails.default.url} /></a>
```

Comment:
I didn't find a prop `"makeActive"` maybe you mean `activatePreview`?
It also better to handle such scenario at onClick handlers